tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -2703,10 +2907,27 @@ public function theFollowingGroupsShouldNotExist(TableNode $table):void {
 	 */
 	public function theseGroupsShouldNotExist(string $shouldOrNot, TableNode $table):void {
 		$should = ($shouldOrNot !== "not");
-		$groups = $this->getArrayOfGroupsResponded($this->getAllGroups());
+		$graphMode = OcisHelper::isTestingWithGraphApi();
+		if ($graphMode) {
+			$groups = $this->graphContext->adminHasRetrievedGroupListUsingTheGraphApi();
```

Comment:
just to make consistent
```suggestion
		$useGraph = OcisHelper::isTestingWithGraphApi();
		if ($useGraph) {
			$groups = $this->graphContext->adminHasRetrievedGroupListUsingTheGraphApi();
```